iothub/device/src/DeviceClient.cs

```diff
@@ -325,6 +325,18 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
         /// <returns>The receive message or null if there was no message until the specified time has elapsed</returns>
         public Task<Message> ReceiveAsync(TimeSpan timeout) => InternalClient.ReceiveAsync(timeout);
 
+        /// <summary>
+        /// Registers a new delegate for receiving a message from the device queue using the default timeout.
```

Comment:
can we use a word "Sets" here

Reply:
I followed the wording that we've used for connection status change callback, method callback etc., so I used "Registers" here.